src/p11_attribute.ml

```diff
@@ -167,9 +162,9 @@ let to_json : type a . a t -> Yojson.Safe.json = fun attribute ->
     | CKA_MODIFIABLE, param ->
         p_bool "CKA_MODIFIABLE" param
     | CKA_EC_PARAMS, param ->
-        p_ec_params "CKA_EC_PARAMS" param
+        p_data "CKA_EC_PARAMS" param
```

Comment:
I was thinking checking that `of_yojson (Assoc [CKA_EC_PARAMS, String "0x4142}")` is `[pack CKA_EC_PARAMS "AB"]`, and similarly for `to_yojson`.